Libraries/Components/TextInput/TextInput.js

```diff
@@ -99,6 +99,15 @@ type Event = Object;
  *   var notMultiline = {
  *     onSubmitEditing: true,
  *   };
+ *
+ * ### TextInput methods
+ *
+ * If you have a ref to the TextInput element, you can invoke several methods
+ * on it to control it:
+ *
+ *  - `setSelectionRange(start, end)` - Sets the start and end positions of the
+ *    current text selection in a TextInput element. Choose the the same value
```

Comment:
I still think that adding a helper to move the cursor is better then docs;

like we discussed in: https://github.com/facebook/react-native/issues/2478#issuecomment-136669384